docs/content/integrate/auth.md

```diff
@@ -22,6 +22,15 @@ openssl rand -base64 32
 
 Example claim:
 
+`HEADER:ALGORITHM & TOKEN TYPE`
```

Comment:
I'm not sure if this make it easier for people to grasp. Would you mind re-writing this as a separate _Generate JWT token_ section on this page? Instead of the `alg` this would then also document that HS256 must be used as signing algorithm. 

Instead of 

```
{{<highlight html>}}
HMACSHA256(
	base64UrlEncode(header) + "." +
	base64UrlEncode(payload),
	your-256-bit-secret
) []secret base64 encoded
{{< /highlight >}}
```
It could describe how to generate a JWT token using jwt.io.